server/views/partials/phaseBanner.njk

```diff
@@ -0,0 +1,12 @@
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+
+{% block phaseBanner %}
```

Comment:
I don't think we need the {% block %} here - we're not overriding this anywhere, so I'm not sure it does anything